pyproject.toml

```diff
@@ -35,7 +35,7 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
 ]
 dependencies = [
-    "apache-airflow>=2.4",
+    "apache-airflow>=2.2.4",
```

Comment:
I am trying to spin Airflow with runtime image 4.2.1 which is being used by customer B and while building the image in the examples Astro project it complains about conflicting Airflow versions. I am able to get Airflow up and running now but facing a failure while running the DAG and the error log is here: https://astronomer.slack.com/archives/C03HRT0MRPD/p1678198413109119?thread_ts=1678198362.881859&cid=C03HRT0MRPD.

We might also want to check if this affects the DBT provider and if yes, we can only change this in the newer Databricks providers repository.